### ISSUE TITLE
Server load

### ISSUE THREAD

**Issue by anvaka** [drolbr/Overpass-API#553]
Hello Team!

Thank you for your hard work and dedication! Having overpass api in open access is like having superpowers!

I recently released this website that renders roads from OSM: https://twitter.com/anvaka/status/1218971717734789120 and it received unusual amount of attention (~70K visitors over past few days). 

Coincidentally I noticed that I'm getting timeouts from Overpass API more frequently than I used to. 

I just wanted to check that I'm not impacting you negatively with my traffic. Is there still more room to accept even more requests?

I'm hitting the following endpoints:

* https://overpass.kumi.systems/api/interpreter
* https://overpass-api.de/api/interpreter
* https://overpass.openstreetmap.ru/cgi/interpreter

Is there a way to check queue size, cpu load, network, number of timeouts, etc. on those backends?

**Comment by mmd-osm** [comment]
In general, it's always a good idea to check the Usage policy for the server resources you're using beforehand: https://wiki.openstreetmap.org/wiki/Overpass_API#Public_Overpass_API_instances 

Servers are typically funded by donations (e.g. by FOSSGIS e.V.) and you're expected to adhere to the respective usage policies outlined on the wiki page. As the code is open source, you're very welcome to run your own instance.

I should also note that Kumi systems is operated by a third party, i.e. you need to ask them in case of any issues with the service.

**Comment by anvaka** [comment]
> The name of the backend is in fact Overpass API, not overpass turbo api.

Yes! My bad. This should be fixed now. Please let me know if you notice this anywhere else!

> you're very welcome to run your own instance

I'm curious, how much does it cost to run an instance?

> check the Usage policy for the server resources you're using beforehand

Totally agree. I read that wiki page a few times. From a client perspective - is there a `json` api similar to https://overpass-api.de/api/status that would let me check before fetching?

**Comment by woodpeck** [comment]
A suitable machine to run your own instance on starts at ~ 65 EUR per month, e.g. https://www.hetzner.de/dedicated-rootserver/ex52-nvme. (Not affiliated with Hetzner but using them myself - among other things for running a commercial Overpass service.)
